scripts/install-elastio.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
```

Comment:
Ugh, I am looking forward to having `elastioup` Rust binary in this repo instead :sweat:

Reply:
...and a bash script to install/uninstall that rust binary? ))

Reply:
@e-kov 
We can just give direct links to compiled binaries for each platform as terraform does: https://www.terraform.io/downloads.html

Reply:
here we only have 1 installation link for all supported distributions. and 3 arguments for uninstall and 2 rare cases. not much to say that it is difficult. don't recall why you say "ugh" )

Reply:
I say "ugh" just when I see bash in general.
Do we support installation on windows?

Reply:
No, Windows doesn't even have the same shell, so the Windows script will always be different.